Section_I/law8.tex

```diff
@@ -47,9 +47,9 @@
 Before a kick-off at the start of the match or extra time 
 
 \begin{itemize}
-\item a coin is tossed and the team that wins the toss decides which goal it will attack in the first half of the match.
-\item the other team takes the kick-off to start the match. 
-\item the team that wins the toss takes the kick-off to start the second half of the match.
+      \item \removed{a coin is tossed and the team that wins the toss decides which goal it will attack} \added{referee randomly decides which team will attack which goal} in the first half of the match.
+      \item \removed{the other} \added{referee randomly decides which} team takes the kick-off to start the match. 
+      \item the team that \removed{wins the toss} \added{is other from team taking kick-off at first half} takes the kick-off to start the second half of the match.
```

Comment:
Use the wording from the virtual rule book here: "the team that was not given kick-off in the first half of the match takes the kick-off to start the second half of the match."